ipaserver/install/server/replicainstall.py

```diff
@@ -132,24 +132,6 @@ def install_krb(config, setup_pkinit=False, pkcs12_info=None, fstore=None):
     return krb
 
 
-def install_ca_cert(ldap, base_dn, realm, cafile, destfile=paths.IPA_CA_CRT):
```

Comment:
By removing this function ansible-freeipa replica install will not work anymore.

It is my understanding that ansible-freeipa deployment role should be updated, right?

Reply:
Yes, the removal of this function is needed for ansible-freeipa to be able to detect when ipa-certupdate needs to be used instead.